### DISCUSSION TITLE
Twitter and GitHub username changed...

### DISCUSSION THREAD

**Discussion by QwertyIsCoding** [is-a-dev/register#1668] [Support] [answered]
I already got my domain but the problem is that my GitHub page and my Twitter usernames are changed. The CNAME is correct because I changed that, but forgot to change the rest. My Twitter is now @QwertyIsCoding and same for GitHub. Is that a problem?

**Comment by phenax** [comment, marked as answer]
Twitter handle is just in case we need to get in touch with you about your subdomain or the service. Github username however, is how we identify you. The username field should be the github username of the account you wish to make changes to the record with.

**Reply by QwertyIsCoding** [reply]
![image](https://user-images.githubusercontent.com/77963043/152761627-5d0fa9ef-d852-40a0-ab8f-57dd2bcb44ec.png)
Ok... so do I make another pull request or do I have to edit it?

**Reply by QwertyIsCoding** [reply]
Sorry for the trouble...

**Reply by phenax** [reply]
Edit your domain json file on the [is-a-dev/register](https://github.com/is-a-dev/register) repo on github. It'll prompt you to create a pr with the changes.

**Reply by QwertyIsCoding** [reply]
Ok, thankyou. I edited it and created the pr.
